### ISSUE TITLE
Net names in schematics and corresponding constants in source code;

### ISSUE THREAD

**Issue by krisjans** [EdgarsMelnalksnis/BareMetalBinaryWatch#6]
In my opinion NET name in schematic must describe primary net function. 
NET names that match specific SOC pin names is non-portable approach. What if you need to change SOC?
Defines for specific GPIO names most probably exist in in manufacturer's header files - if you use them as NET names, than re-using those names in source code may lead to name clashes. And again - if you change SOC, then those constants won't make sense anymore.

**Comment by EdgarsMelnalksnis** [comment]
I agree, but it`s hard to think better name in this schematic. For example PA7 can be in any row or column and also anode or cathode. Other nets describe primary function in schematic.

**Comment by krisjans** [comment]
main function is LED CONTROL. How about  `LED_CTRL_0` - `LED_CTRL_5` ?
In source code you use `LED_PIN_#` - also not bad. A bit confusing tough - such name implies connection to single LED.
